src/tree/render.js

```diff
@@ -143,45 +89,17 @@ const renderTree = ({ element, dataTree, layout, Theme }) => {
 
   const divBox = select(element)
     .selectAll('div')
-    .data(entries(orientations))
+    .data([{}])
```

Comment:
Is attaching data necessary if it is empty?

Reply:
Tried without it, didn't work.